build/src/main.rs

```diff
@@ -79,17 +86,48 @@ fn main() {
 			.help("Injects RUNTIME_VERSION global export")
 			.takes_value(true)
 			.long("runtime-version"))
+		.arg(Arg::with_name("source_target")
+			.help("Cargo target type kind (wasm32-unknown-unknown/emscripten)")
```

Comment:
`wasm32-unknown-unknown`/`wasm32-unknown-emscripten` ?

Reply:
it's the same but shorter